experimental/BasisLieHighestWeight/src/UserFunctions.jl

```diff
@@ -570,3 +570,180 @@ function basis_coordinate_ring_kodaira_ffl(
     V, degree, operators, monomial_ordering
   )
 end
+
+@doc raw"""
+    basis_lie_demazure(type::Symbol, rank::Int, highest_weight::Vector{Int}, weyl_group_elem::Vector{Int}; monomial_ordering::Symbol=:degrevlex)
+    basis_lie_demazure(type::Symbol, rank::Int, highest_weight::Vector{Int}, weyl_group_elem::Vector{Int}, birational_sequence::Vector{Int}; monomial_ordering::Symbol=:degrevlex)
+    basis_lie_demazure(type::Symbol, rank::Int, highest_weight::Vector{Int}, weyl_group_elem::Vector{Int}, birational_sequence::Vector{Vector{Int}}; monomial_ordering::Symbol=:degrevlex)
+
+Compute a monomial basis for the demazure module with extremal weight
+`highest_weight * weyl_group_elem` (in terms of the fundamental weights $\omega_i$),
+for a simple Lie algebra of type `type_rank`.
+
+If no birational sequence is specified, all operators in the order of `basis_lie_highest_weight_operators` are used.
+A birational sequence of type `Vector{Int}` is a sequence of indices of operators in `basis_lie_highest_weight_operators`.
+A birational sequence of type `Vector{Vector{Int}}` is a sequence of weights in terms of the simple roots $\alpha_i$.
+
+`monomial_ordering` describes the monomial ordering used for the basis.
+If this is a weighted ordering, the height of the corresponding root is used as weight.
+
+warning!!! The demazure module is twisted and the results need to be interpreted accordingly.
```

Comment:
```suggestion
!!! warning
    The demazure module is implemented as a submodule of a twisted highest weight module.
    The results of this function need to be interpreted accordingly.
```

This formats the warning nicely, see https://documenter.juliadocs.org/stable/showcase/#Warning-admonition for details.
And please also add this to all the other docstrings